src/index.html

```diff
@@ -11,6 +11,50 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <h1 hidden>Search bar airbnb</h1>
```

Comment:
The main heading is hidden. Consider removing the 'hidden' attribute if you want it to be visible.

Reply:
why did you add hidden h1?
delete tag or attribute hidden

Reply:
The main heading shouldn't render on page. Test isn't passed so I set attribute 'hidden'.